hieraserver/rest.go

```diff
@@ -94,6 +98,7 @@ func startServer(cmd *cobra.Command, _ []string) {
 			}
 			opts.Merge = params.Get(`merge`)
 			opts.Type = params.Get(`type`)
+			opts.Variables = params[`var`]
```

Comment:
Looks like this will override any --var options that were given when the server was started. Perhaps do:
```golang
opts.Variables = append(opts.Variables, params[`var`]...)
```

Reply:
Good point, updated